elements/pfe-button/README.md

```diff
@@ -14,14 +14,6 @@ the only light DOM child of `pfe-button`.
 </pfe-button>
 ```
 
-### Size
-
-```html
-<pfe-button size="large">
-  <button>Large Button</button>
-</pfe-button>
-```
-
 ### Accessibility
 `<pfe-button>` addresses the issues associated with typical implementations of
 web component buttons. When using a `<pfe-button>` in a `<form>` element, the
```

Comment:
The light dom element will only pass `id | class | type` attributes to the shadow DOM should the README.md and subsequent docs reflect this restriction?  Would we want to open this up to all valid attributes for button in the future i.e `data` or `value`?

Reply:
this is taken care of in the latest iteration - we no longer copy the button into shadow dom